docs/reference/buildx_dial-stdio.md

```diff
@@ -0,0 +1,49 @@
+# docker buildx dial-stdio
+
+<!---MARKER_GEN_START-->
+Dial stdio
+
+### Options
+
+| Name         | Type     | Default | Description                                      |
+|:-------------|:---------|:--------|:-------------------------------------------------|
+| `--builder`  | `string` |         | Override the configured builder instance         |
+| `--platform` | `string` |         | Target platform: this is used for node selection |
+| `--progress` | `string` | `quiet` | Set type of progress output (auto, plain, tty).  |
+
+
+<!---MARKER_GEN_END-->
+
+## Description
+
+dial-stdio uses the stdin and stoud streams of the command to proxy to the configured builder instance.
+It is not intended to be used by humans, but rather by other tools that want to interact with the builder instance API.
+
+## Examples
+
+Example go program that uses the dial-stdio command wire up a buildkit client.
+This is for example use only and may not be suitable for production use.
+
+```go
+c1, c2 := net.Pipe()
+cmd := exec.Command("docker", "buildx", "dial-stdio")
+cmd.Stdin = c1
+cmd.Stdout = c2
+
+if err := cmd.Start() {
+    c1.Close()
+    c2.Close()
+    return nil, err
+}
+
+defer cmd.Wait()
+
+go func() {
+    cmd.Wait()
+    c2.Close()
+}()
+
+client.New(ctx, "", client.WithContextDialer(func(ctx context.Context, s string) (net.Conn, error) {
+    return c2, nil
```

Comment:
Why use static instance here? I think it should invoke new command when dialer gets called.

```
func dial(...) (net.Conn, error) {
  _ = exec.Command()
  ...
}

client.New(ctx, "", client.WithContextDialer(dial))
```